docs/v2-system-addon/geo_locale.md

```diff
@@ -0,0 +1,20 @@
+# Setting custom `geo`, `locale`, and update channels
+
+There are instances where you may need to change your local build's locale, geo, and update channel (such as changes to the visibility of Discovery Stream on a per-geo/locale basis in `ActivityStream.jsm`).
+
+## Changing update channel
+
+- Change `app.update.channel` to desired value (eg: `release`) by editing `LOCAL_BUILD/Contents/Resources/defaults/pref/channel-prefs.js`. (**NOTE:** Changing pref `app.update.channel` from `about:config` seems to have no effect!)
+
+## Changing geo
+
+- Set `browser.search.region` to desired geo (eg `CA`)
+
+## Changing locale
+
```

Comment:
Let's add a note here to emphasize that those prefs are configurable only on nightly or local build.